cmd/clusterctl/cmd/upgrade_apply.go

```diff
@@ -85,6 +86,8 @@ func init() {
 		"IPAM providers and versions (e.g. infoblox:v0.0.1) to upgrade to. This flag can be used as alternative to --contract.")
 	upgradeApplyCmd.Flags().StringSliceVar(&ua.runtimeExtensionProviders, "runtime-extension", nil,
 		"Runtime extension providers and versions (e.g. test:v0.0.1) to upgrade to. This flag can be used as alternative to --contract.")
+	upgradeApplyCmd.Flags().StringSliceVar(&ua.addonProviders, "add-on", nil,
```

Comment:
```suggestion
	upgradeApplyCmd.Flags().StringSliceVar(&ua.addonProviders, "addon", nil,
```
To be consistent with flags in the other commands.

Reply:
And similarly other places in the e2e tests as well.